src/main/java/com/xiaomi/infra/pegasus/client/PegasusClient.java

```diff
@@ -43,17 +43,19 @@ public long hash(byte[] key) {
   }
 
   private PegasusTable getTable(String tableName) throws PException {
-    return getTable(tableName, 0);
+    return getTable(tableName, 0, false);
```

Comment:
```suggestion
    TableOptions opts = new TableOptions();
    return getTable(tableName, opts);
```

For extensibility, this way is preferred.